build/mvn

```diff
@@ -187,6 +187,8 @@ if [ ! -z "${SPARK_LOCAL_IP}" ]; then
   echo "Using SPARK_LOCAL_IP=$SPARK_LOCAL_IP" 1>&2
 fi
 
+export USE_MAVEN=1
```

Comment:
Only Maven will trigger this issue, so add an environment variable needs to identify Maven test